crest/Assets/Crest/Crest/Scripts/Spline/SplinePoint.cs

```diff
@@ -63,6 +68,14 @@ public override void OnInspectorGUI()
                 return;
             }
 
+            if (parent.TryGetComponent(out ISplinePointCustomDataSetup customData))
+            {
+                if (customData.AttachDataToSplinePoint(thisSP.gameObject))
+                {
+                    EditorUtility.SetDirty(thisSP.gameObject);
+                }
+            }
```

Comment:
Ah good point, i was going for something unorthodox here - I decided to make the code only weakly dependent on the data being present.

So if the component is not attached to a spline point, the default flow will be used. The mesh construction should handle this gracefully. If i want to change the flow somewhere, i click the spline point and the component attaches.

This is maybe not intuitive, or maybe a bit odd. Or it may fail in some weird ways. Let me know what you think.

Reply:
That makes sense. It's almost how Unity does it with the additional data components. So the issue really was that the default flow wasn't working? When I opened the scene there was no flow happening even though the flow input had a speed of 5.

Reply:
Oops there was indeed no default value when the data is not attached. It should be fixed now (defaults to 1, which is then multiplied by flow setting on material)